scanner/Makefile

```diff
@@ -165,6 +161,19 @@ copy-scripts: $(addprefix ../image/rhel/static-bin/,$(SCRIPTS))
 	@echo "+ $@"
 	$(SILENT)cp $^ image/scanner/scripts
 
+#################
+## Integration ##
+#################
+
+integration-d       := matcher/metadata/postgres
+integration-go-tag  := scanner_integration
```

Comment:
used `scanner_integration` instead of `integration` to make sure it doesn't collide with `integration` as used outside the `scanner` directory

Reply:
Nitpick: Maybe a Makefile comment instead of a PR comment.